docker/tensorflow/mivisionx-with-tensorflow.dockerfile

```diff
@@ -55,4 +55,8 @@ RUN git clone https://github.com/GPUOpen-ProfessionalCompute-Libraries/MIVisionX
 RUN cd ~/models/research/ && \
 protoc object_detection/protos/*.proto --python_out=. && \
 cp object_detection/packages/tf2/setup.py . && \
-pip3 install --use-feature=2020-resolver .
+python -m pip install .
```

Comment:
Is this changed needed as part of this PR?

Reply:
Shobana's fork is not updated to the latest ToT so this is a change from ToT that has been added to my PR's changes